ext/fetch/26_fetch.js

```diff
@@ -352,10 +351,11 @@ function httpRedirectFetch(request, response, terminator) {
  */
 function fetch(input, init = { __proto__: null }) {
   let span;
+  let context;
   try {
     if (TRACING_ENABLED) {
-      span = new Span("fetch", { kind: 2 });
-      enterSpan(span);
+      span = builtinTracer().startSpan("fetch", { kind: 2 });
```

Comment:
Not important for this PR, but could we use some sort of enum object for the kind? The numerical value is not super informative.

Reply:
It is an enum in the user API. I'll see about adding it to the internal api too